### PR TITLE
MOM 11 cards

### PR DESCRIPTION
**Pull request by Simisays**
This should wrap up the remaining MOM cards, nothing too hard here

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/realmbreakers_grasp.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Name:Realmbreaker's Grasp
+ManaCost:1 W
+Types:Enchantment Aura
+K:Enchant artifact or creature
+A:SP$ Attach | ValidTgts$ Artifact,Creature, | TgtPrompt$ Select target artifact or creature | AILogic$ Curse
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/realmbreakers_grasp.txt:5]: extra "," after creature?

File: forge-gui/res/cardsfolder/upcoming/order_of_the_mirror_order_of_the_alabaster_host.txt [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+Oracle:{3}{W/P}: Transform Order of the Mirror. Activate only as a sorcery. ({W/P} can be paid with either {W} or 2 life.)
+
+ALTERNATE
+
+Name:Order of the Alabaster Host
+ManaCost:no cost
+Colors:white,blue
+Types:Creature Phyrexian Knight
+PT:3/3
+T:Mode$ AttackerBlockedByCreature | ValidCard$ Card.Self | ValidBlocker$ Creature | Execute$ TrigPump | TriggerDescription$ Whenever CARDNAME becomes blocked by a creature, it deals 2 damage to that creature.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/order_of_the_mirror_order_of_the_alabaster_host.txt:16]: Description doesn’t match

File: forge-gui/res/cardsfolder/upcoming/seed_of_hope.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Name:Seed of Hope
+ManaCost:G
+Types:Instant
+A:SP$ Mill | NumCards$ 2 | RememberMilled$ True | SubAbility$ DBChangeZone | SpellDescription$ Mill two cards. You may put a permanent card from among the milled cards into your hand. You gain 2 life. (To mill two cards, put the top two cards of your library into your graveyard.)
+SVar:DBChangeZone:DB$ ChangeZone | Hidden$ True | Origin$ Graveyard,Exile | Destination$ Hand | ChangeType$ Card.IsRemembered+Permanent | SelectPrompt$ You may put a permanent card from among the milled cards into your hand. | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/seed_of_hope.txt:6]: Lifegain?

File: forge-gui/res/cardsfolder/upcoming/seraph_of_new_capenna_seraph_of_new_phyrexia.txt [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+ALTERNATE
+
+Name:Seraph of New Phyrexia
+ManaCost:no cost
+Colors:white,black
+Types:Creature Phyrexian Angel
+PT:3/3
+K:Flying
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigPump | OptionalDecider$ You | TriggerDescription$ Whenever CARDNAME attacks, you may sacrifice another creature or artifact. If you do, Seraph of New Phyrexia gets +2/+1 until end of turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/seraph_of_new_capenna_seraph_of_new_phyrexia.txt:18]: CARDNAME

File: forge-gui/res/cardsfolder/upcoming/seed_of_hope.txt (hunk 3 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/seed_of_hope.txt:6]: Still not handled?

**Review by tool4ever — APPROVED** [review]

**Pull request merged**